sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3972,8 +3972,8 @@ def hpat_pandas_series_idxmin(self, axis=None, skipna=None):
 
     Limitations
     -----------
-    Parameter ``axis`` is supported only with default value ``None``.
-    Parameter ``skipna`` cannot be ``False`` with data of string type.
+    - Parameter ``axis`` is supported only with default value ``None``.
```

Comment:
Let's be more consistent and write either "is supported only with default value" or 'not supported'